Tools/debug/CollectSystemInfo.ps1

```diff
@@ -0,0 +1,234 @@
+#  are met:
+#  1. Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+#  2. Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#  3. Neither the names of the copyright holders nor the names of their contributors
+#     may be used to endorse or promote products derived from this software
+#     without specific prior written permission.
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+#  ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE LIABLE
+#  FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+#  DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+#  OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+#  HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+#  LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+#  OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+#  SUCH DAMAGE.
+
+
+#  Ensure the script runs with an unrestricted execution policy (for Windows 10 and Windows Server 2016)
+#  Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process -Force
+
+#  For gathering event logs run the script as an administrator
+
+#  IncludeSensitiveData is used to include memory dumps add this parameter to your command line to collect them
+#  Example:  .\CollectSystemInfo.ps1 -IncludeSensitiveData
+
+param (
+    [switch]$IncludeSensitiveData
+)
+
+function Get-SystemConfiguration {
+    try {
+        Start-Process -FilePath "msinfo32.exe" -ArgumentList "/report", "msinfo32.txt" -Wait
+    } catch {
+        Write-Warning "Failed to collect system configuration: $_"
+    }
+}
+
+function Get-EventLogs {
+    try {
+        $logNames = @("system", "security", "application")
+        foreach ($logName in $logNames) {
+            wevtutil epl $logName "$logName.evtx"
+            wevtutil al "$logName.evtx"
+        }
+    } catch {
+        Write-Warning "Failed to collect event logs: $_"
+    }
+}
+
+function Get-DriversList {
+    if (-not ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)) {
+        Write-Warning "This function requires administrative privileges. Please run the script as an administrator."
+        return
+    }
+    try {
+        $drivers = Get-WindowsDriver -Online -All
+        $drivers | Select-Object -Property * | Export-Csv -Path drv_list.csv -NoTypeInformation
```

Comment:
There is no point of having a variable for this. You can do the same with `Get-InstalledApplications`